### DISCUSSION TITLE
How do i enable SET allow_experimental_object_type=1 globally

### DISCUSSION THREAD

**Discussion by khanakia** [ClickHouse/ClickHouse#63785] [Q&A]
I am using Atlasgo migration tool for click house and they do not allow me to send the allow_experimental_object_type along with the migration command.

So how do i enable `SET allow_experimental_object_type=1` globally

because I get this error currently

```
Error: code: 36, message: The change of data type String of column `body` to Object('json') is not allowed. It has known bugs
```

**Comment by Algunenano** [comment]
That error can't be fixed by changing a setting. It's preventing known bugs (crashes) with Object('json') that won't be fixed.

You should not use `Object('json')`

**Comment by khanakia** [comment]
I am building a logging system and I want to store the request and response body. So is it ok to save it as string?

If yes also how will be able to filter the data as response body can have dynamic keys and values

**Reply by Algunenano** [reply]
In the short term you can use String and the JSON functions (https://clickhouse.com/docs/en/sql-reference/functions/json-functions). In the medium term, the team is working on a different type without the limitations of the Object('json') type (https://github.com/ClickHouse/ClickHouse/issues/54864)
